pysipp/agent.py

```diff
@@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
     configuration options for a SIP UA.
     """
     # we skip `error` since we can get it from stderr
-    _log_types = 'screen calldebug message log'.split()
+    _log_types = 'screen calldebug message log error'.split()
```

Comment:
@fchareyron You'll have to post the full traceback as I'm not entirely sure what's going wrong. Also note you can syntax highlight code blocks here in Github comments using [these instructions](https://help.github.com/articles/creating-and-highlighting-code-blocks/).

Basically what we want to see (in order to get your changes into the projects) is that the unit test suite passes. If you need to modify the tests that is fine as long as it makes sense and is reliable.